sample-apps/discount-functions-sample-app/extensions/product-discount/src/api.rs

```diff
@@ -10,107 +8,69 @@ pub type ID = String;
 pub mod input {
     use super::*;
     use serde::Deserialize;
-    pub type UnsignedInt64 = u64;
 
     #[derive(Clone, Debug, Deserialize)]
     #[serde(rename_all(deserialize = "camelCase"))]
     pub struct Input {
-        pub customer: Option<Customer>,
-        pub delivery_lines: Option<Vec<DeliveryLine>>,
-        pub locale: Option<String>,
-        pub merchandise_lines: Option<Vec<MerchandiseLine>>,
-        pub discount_node: Option<DiscountNode>,
```

Comment:
Should we give them a fully generated API file? The compiler will strip out whatever they don't use

Reply:
Was having trouble with serde serializing `oneOf` but I'm not sure if that's resolved